pyproject.toml

```diff
@@ -66,7 +66,7 @@ flake8-docstrings = "^1.7.0"
 moviepy = "1.0.3"
 python-levenshtein = "^0.21.1"
 magic-wormhole = "0.12.0"
-pyside6 = "^6.5.1.1"
+pyside6 = "6.7.1"
```

Comment:
I had to upgrade this as when I ran`pytest` it was failing due to pyside6 being not at correct version.